notes.md

```diff
@@ -331,6 +375,69 @@ it as 0x53B9 instead of 0x53B8. `Matroska Readers` may support these legacy file
 Matroska v2 or 0x53B9. The older values were 0: mono, 1: right eye, 2: left eye, 3: both eyes.
 
 
+# Default track selection
+
+This section provides example algorithms that a `Matroska Player` **MAY** use to select
+default audio and subtitle tracks. These algorithms rely on the following settings being
+available to the user:
+
+- Preferred audio languages (zero or more language codes)
+- Preferred subtitle languages (zero or more language codes)
+- Prefer original-language audio (bool)
+- Display captions (bool)
+- Hearing-impaired (bool)
+- Visually-impaired (bool)
```

Comment:
- select commentaries (bool)

Most users would never do that but in some professional circles it may be useful.

Reply:
I'm not sure this is a reasonable feature to attempt to provide, since there may be multiple entirely-different commentary tracks (and I don't think we really want to get into the nitty-gritty details of, say, director's commentary vs cast commentary vs translator's commentary vs [etc etc etc], that seems like more of a job for the track title). If a player does want to provide this, they're free to extend the functionality as they please.

Reply:
OK, but as said above we should at least define which elements are involved to pick the right track on startup (then the user can select whatever else). Two tracks with the same flags except for the commentary one should be picked in most cases. So it may be an option that noone ticks but at least the player should know that's it's preferred.